spec/requests/admin/comments_spec.rb

```diff
@@ -58,6 +58,8 @@ def comment_in_context_path( comment )
     end
 
     it 'deletes the selected comments if you say they are spam' do
+      skip 'Valid Akismet API KEY required' if ENV[ 'AKISMET_API_KEY' ].blank?
```

Comment:
These `skip`s look like candidates for being replaced by VCR as described here: https://github.com/denny/ShinyCMS-ruby/pull/293#discussion_r413919191